components/messenger.rst

```diff
@@ -68,6 +68,34 @@ Concepts
    to use for transport, markers identifying a received message or any sort of
    metadata your middleware or transport layer may use.
 
+**Message**
+   There is no specific requirement for a message, it can be any kind of object except it
+   should be serializable and unserializable by a Symfony Serializer instance.
```

Comment:
Actually not. Native serialization mechanism [is used by default](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L1222-L1224)